quodlibet/quodlibet/config.py

```diff
@@ -215,6 +215,9 @@ def _config_text(text):
         "spaces": "false",
         "windows": "true",
         "ascii": "false",
+        "moveart": "false",
```

Comment:
`move_art` and `move_art_overwrite` seem better IMO

Reply:
done